infra/prod/_modules/storage/storage_account_fims.tf

```diff
@@ -9,7 +9,28 @@ module "storage_account_fims" {
   resource_group_name           = var.resource_group_name
   location                      = var.location
   advanced_threat_protection    = true
-  public_network_access_enabled = false
+  public_network_access_enabled = true
```

Comment:
why?

Reply:
@Krusty93 it's a temp fix (that will be reverted once applied), because this template make uses of GitHub runners that does not have access to our private resources! 

this issue will be fixed with https://github.com/pagopa/dx-typescript/pull/14